doc/manual.md

```diff
@@ -121,7 +121,16 @@ We can also get a shell into them
 $ gojira shell -t 1.2.0
 [kong-1.2.0:/kong]# exit
 $ gojira shell
-[kong-master:/kong]#
+[kong-master:/kong]# exit
+```
+
+We can pass Environment variable bindings into the kong container's shell
+
+```
+$ gojira shell --env FIZZ=buzz
+[kong-master:/kong]# echo $FIZZ
+buzz
+[kong-master:/kong]# exit
```

Comment:
Being at it, it would be cool to also note that the simple docker form is also supported:

```
$ export FIZZ=buzz
$ gojira shell --env FIZZ
> echo $FIZZ
buzz
```

Reply:
@eskerda that's grand! I documented this in https://github.com/Kong/gojira/pull/10/commits/7072eb92c330db09aacf286413b00311f16d3fb9